backend/src/main.cpp

```diff
@@ -258,6 +258,25 @@ void SetCid(const Napi::CallbackInfo& info)
     UserSession::cid = cid;
 }
 
+void SetRadioEffects(const Napi::CallbackInfo& info)
+{
+    auto radioEffects = info[0].As<Napi::String>().Utf8Value();
+    std::transform(radioEffects.begin(), radioEffects.end(), radioEffects.begin(), std::tolower);
+    auto enableInputFilters = true;
+    auto enableOutputEffects = true;
+
```

Comment:
This still suffers from the same issue as the previous version, just in reverse. Now the "on" state isn't explicitly handled anywhere, and unknown states will turn audio effects on.

Why not something like this:

```c++
bool enableInputFilters;
bool enableOutputFilters;

if (radioEffects == "off")
{
      // do stuff;
}
else if (radioEffects == "on)
{
    // do stuff;
}
else if (...) // Add another else if for each possible radioEffects value
{
}
else {
   // Log an error
   return;
}
```

This makes it abundantly clear to future people working on this code what the valid options are, doesn't magically default to something if the passed in string isn't supported, and logs an error so people in the future who passed unsupported values can chase down what they did wrong.

Reply:
SetHardwareType does the same. It uses a default type, if a not handled case is selected.

Reply:
I'll defer to @pierr3 on this one then. IMHO just because SetHardwareType does it that way doesn't make it good 😂

Reply:
Done.

Reply:
The two variables have to be `bool` instead of `auto` since they don't have an initializer. Are you remembering to do an `npm run build:backend` before pushing the changes? This doesn't build locally on my machine because of this.